src/components/Cart/CartList.jsx

```diff
@@ -0,0 +1,123 @@
+import "./CartList.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { CartListCard } from "../Card/CartListCard";
+import { CartBill } from "./CartBill";
+
+const CartList = () => {
+    const [showEmptyCart, setShowEmptyCart] = useState(false);
+    const [cartListItemsCount, setCartListItemsCount] = useState(0);
+
+    // Initialize CartList Items Count
+    useEffect(() => {
+        const initialCartListItemsCount = document.querySelectorAll(".cart-cards .card").length;
+        setCartListItemsCount(initialCartListItemsCount);
+    }, []);
+
+    // Show Empty CartList if items count is zero
+    useEffect(() => {
+        !cartListItemsCount ? setShowEmptyCart(true): setShowEmptyCart(false);
+    }, [cartListItemsCount]);
+
+    // User Cart Data will be fetched from the API.
+    const cartItems = [
```

Comment:
cart items should be fetched from db

Reply:
Yeah, once I integrate the API for fetching cart items, there will be no need of hardcoding it 😅